assignments/arrays.js

```diff
@@ -82,13 +359,14 @@ console.log();
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
-let oldCars = [];
-console.log(); 
+const getAge = []
+for (let i = 0; i < inventory; i++) {
+    if (inventory[i].id > 2000)
```

Comment:
The [if statement] needs to be adjusted to include the appropriate field and the would need to evaluate as less than 2000 to select the older cars.